www/src/components/ecosystem/ecosystem-section.js

```diff
@@ -105,14 +99,14 @@ const EcosystemSection = ({
   featuredItems,
   className,
 }) => (
-  <EcosystemSectionRoot className={className}>
+  <section sx={ecosystemSectionRootStyles} className={className}>
```

Comment:
I don't think the `className` prop is used anywhere? Can probably just delete.

Reply:
👍

Reply:
Yep, but need `{...rest}` still for https://github.com/gatsbyjs/gatsby/blob/c1368c06fb975bd792ebb8f9d0c5a5e4ebcba388/www/src/components/homepage/homepage-ecosystem.js#L30-L45

Reply:
Btw. we lost those icons on the "Starters" and "Plugins" cards I think?

![image](https://user-images.githubusercontent.com/21834/85920245-6b0cb980-b872-11ea-9643-dc8da59f2ff0.png)